stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/SyncStatus.kt

```diff
@@ -8,7 +8,9 @@ public enum class SyncStatus(public val status: Int) {
     /** after the retry strategy we still failed to sync this */
     FAILED_PERMANENTLY(2),
     /** when sync is in progress */
-    IN_PROGRESS(3);
+    IN_PROGRESS(3),
+    /** when message waits its' attachments to be sent */
+    WAIT_ATTACHMENTS(4);
```

Comment:
```suggestion
    AWAITING_ATTACHMENTS(4);
```
Because **_awaiting_** something = _**waiting for**_ something